CMakeLists.txt

```diff
@@ -110,15 +124,18 @@ else()
     find_package(Boost COMPONENTS system)
     target_link_libraries(boost_json
         PUBLIC
+            Boost::container
```

Comment:
If this is going to link to `container`, the `find_package` probably also needs to be adjusted.

Reply:
These are not libraries that find_package would find. I assume they are created by the build super-project (I found the file like this)